.gitignore

```diff
@@ -1,3 +1,15 @@
 node_modules
 npm-debug.log
 tmp
+.idea/.name
+.idea/compiler.xml
+.idea/copyright/profiles_settings.xml
+.idea/encodings.xml
+.idea/inspectionProfiles/Project_Default.xml
+.idea/inspectionProfiles/profiles_settings.xml
+.idea/misc.xml
+.idea/modules.xml
+.idea/vcs.xml
+.idea/workspace.xml
+grunt-xgettext.iml
+.idea/dictionaries/kguo.xml
```

Comment:
Just add .idea/ and *.iml to the .gitignore and then the intelliJ settings/project will be ignored completely.